src/Generator/Process/Processor.cs

```diff
@@ -362,6 +363,7 @@ private static List<RemoveEntryInfo> GetRemoveEntries(List<Feature> features, GL
                                 function.NativeFunction.EntryPoint,
                                 "",
                                 Array.Empty<ParameterDocumentation>(),
+                                "",
```

Comment:
We should add a TODO here to add a link to the extension spec file this function was introduced in.